tox.ini

```diff
@@ -14,10 +15,13 @@ usedevelop=True
 commands=
     core: py.test {posargs:tests/}
     doctest: py.test --doctest-glob='docs/*.rst' --doctest-modules docs rlp
+    core-fallback: pip uninstall -y rusty-rlp
```

Comment:
That's probably not the supposed way of handling it but it works. Naturally I first tried `pip uninstall -y rusty-rlp && py.test {posargs:tests/}` but `tox` doesn't allow that. Then I noticed that the uninstall is already enough because (for some reason) it still derives the `py.test {posargs:tests/}` from `core` so it does what it is supposed to. I'm very open to learn about cleaner alternatives :D

Reply:
Yeah, the prefixes match on substrings, so the `core: py.test {posargs:tests/}` line still matches `core-fallback` environments. I'm not clear on how ordering works, if it's even stable. My read of the config file is that pytest would run before the pip uninstall, but that's based on intuition, not a careful reading of the tox.ini spec.

Reply:
Ah that explains it. Regarding ordering. I don't know either but it appeared to work (I tested it with intentional making it fail and ultimately also with the log message that I introduced). I could also try to make it fail if the log message isn't in the output so that we can be sure it fails loudly if there actually is an issue with that approach.